packages/colorpickers/src/styled/ColorPicker/StyledSliders.ts

```diff
@@ -15,6 +15,29 @@ interface IStyledSlidersProps {
   $isOpaque?: boolean;
 }
 
+const sizeStyles = (props: IStyledSlidersProps & ThemeProps<DefaultTheme>) => {
+  if (props.$isOpaque) return undefined;
```

Comment:
```suggestion
  if (props.$isOpaque) {
    return undefined;
  }
```

Conditions in Garden should always be blocks. We used to lint for this, but [the rule](https://eslint.org/docs/latest/rules/brace-style) was deprecated. Probably need to bring it back with a plugin since prettier doesn't cover us.